tests/test.nim

```diff
@@ -1,6 +1,6 @@
 import testes
 
-import gram
+import ../gram
```

Comment:
We need to test that it's found correctly in the environment, so this should return to an unqualified import.  `testes` knows how to build the test with a proper `--path` statement, and the user's package manager will handle adding it to a site-specific `nim.cfg`.